src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -87080,7 +87080,7 @@ void Tests()
 "
             );
 
-            AssertTryGetSpeculativeSemanticModel(source, NullableContextOptions.Enable, type, "A<System.String>");
+            AssertTryGetSpeculativeSemanticModel(source, NullableContextOptions.Enable, type, "A<System.String>!");
```

Comment:
This doesn't seem right? Nullable should be disabled, we shouldn't have a `!` on the end.

Reply:
> This doesn't seem right? Nullable should be disabled, we shouldn't have a ! on the end.

The analysis is done any way. In any case this is simply translating what SemanticModel calculates and the calculation itself is out of scope of this PR. 

---
In reply to: [357763457](https://github.com/dotnet/roslyn/pull/40338#discussion_r357763457) [](ancestors = 357763457)